Makefile

```diff
@@ -29,7 +29,7 @@ TAGS      :=
 LDFLAGS   := "-w -s -X 'main.version=${VERSION}'"
 REGISTRY ?= k8scloudprovider
 
-ifneq ("$(DEST)", "$(PWD)")
+ifneq ("$(realpath $(DEST))", "$(realpath $(PWD))")
```

Comment:
`realpath` is not available on MacOS by default. Please add checks to ensure that `realpath` exists before using, and put a fall back in place if `realpath` is not available.

Reply:
I've checked that it works with `GNU Make 3.81` that is installed by default on MacOS (or rather with DevTools). `realpath` here is not an external binary, it's an internal Make command.